### ISSUE TITLE
"off" buzzer sounds

### ISSUE THREAD

**Issue by zerkman** [zerkman/zest#9]
in the Flip-o demo main tune, the bass buzzer effect does not sound right.

**Comment by zerkman** [comment]
The tune is "Middle Earth Theme" by Jess. Correct sounding version can be heard here: https://www.youtube.com/watch?v=UenaZp-QDjg

**Comment by Gunstick** [comment]
Thanks for link to my register dump :-) Is middle earth theme also sounding wrong when played stand-alone? If so, then the register write sequence seems to be different from all other tunes. You have a recording of the "wrong" flip-o-demo?

**Comment by Gunstick** [comment]
My routine (or sc68) is probably optimizing the display, so the YM initialization is not shown. I suppose it's writing $0000 to envelope.
I see some odd behaviour: the envelope is set to $0000 (high speed) every time it is not playing.

**Comment by zerkman** [comment]
Actually the problem was not spotted by me, but people watching a live stream by @ggnkua on Twitch :)
I made changes to audio output since then, so maybe that fixed it.

**Comment by zerkman** [comment]
Here's a video of Flip-o demo running on the current version of zeST : https://tube.sector1.fr/w/mucRnC19Vrtz57Gv4FiiKP

I haven't noticed any problems in the sound quality but my hearing is not perfect.

**Comment by zerkman** [comment]
I don't see/hear any problem, so closing the issue. Anyone feel free to provide further input if you think the problem persists.

**Comment by Gunstick** [comment]
Hi

just from memory, the buzzer sound directly at the start, about the 4th note, is tiny slightly different.
Not from the frequency, but from the timbre.
But that could also be a typical hardware effect of differently synched ST and YM.
For the rest of the demo all is perfect.
I think I noticed one stutter, but I did not check if that's the video player, my PC or the zeST.

So roll like that, it looks really great.
